### ISSUE TITLE
Remove the labeling rectangle in last frame from a video file cause all previous labeling rectangles go away

### ISSUE THREAD

**Issue by shaojun** [openvinotoolkit/cvat#2999]
### Expected Behaviour
the last several frames does not contain the target object, so I select the labeling rectangle and delete it, the previous frames with target object SHOULD still keep the labeling rectangles.
Or at least prompt to warn user this operation will clear all previous labeling rectangles. 
### Current Behaviour
the labeling rectangles in previous frames are all gone.

### Your Environment
Server version: 1.2
Core version: 3.10.0
Canvas version: 2.2.1
UI version: 1.13.3

This is a disaster for me that hundreds of previous labelled frames are all lost labeling rectangles, without any prompt.

**Comment by nmanovic** [comment]
@shaojun , probably you are right. For tracks the delete operation should mark a frame as outside by default.

**Comment by shaojun** [comment]
@nmanovic thanks for this great repo and quick response here, if this behavior is confirmed, then I'm personally think this is a `high priority bug` rather than a `enhancement`, reason:

-  can cause whole task's labeling data lose silently.
-  easy to trigger by ordinary user operations.

**Comment by nmanovic** [comment]
@shaojun , the current behavior was for years. Users can always undo operations if they did it by an accidents. But I already heard several complains about the behaviours.

**Comment by shaojun** [comment]
@nmanovic this is not by user accident, it's on purpose operation since some frames may not contain the target object.

Could you suggest for current version, how to remove labeling rectangles for target-object-not-exists frames?

**Comment by nmanovic** [comment]
@shaojun , if my guess is correct and you are annotating a video file, you need to mark the rectangle as outside. See documentation for more details: https://github.com/openvinotoolkit/cvat/blob/develop/cvat/apps/documentation/user_guide.md

**Comment by shaojun** [comment]
@nmanovic thanks for the point, what I'm doing is uploading a video file which taken by my phone and then try labeling all target objects in each frames and finally export to `VOC format` (each frame will be a single JPEG file) for object detection training purpose.  
(I take the video by phone rather than pictures is because I think take pictures by phone is taking more time)

please help to confirm, with the `track mode`, when the object disappeared in the middle of serials of frames, I should click the `switch outside property` to dis-continue the track rectangle shown from previous frames, and then repeat `Go Next` until the frame of object show again, then I should draw another `track`, is this correct?

**Comment by zhiltsov-max** [comment]
@shaojun, yes, `outside` works exactly the way you describe. You'll probably need to enable display of invisible annotations in the UI settings to select and continue the track when the object reappears in the video.

**Comment by shaojun** [comment]
@zhiltsov-max thanks. could u clarify a bit about the purpose of the merge track, as i think the 3 steps: draw track-> outside property -> redraw another track already cover case.

**Comment by zhiltsov-max** [comment]
To support other way to create a single track. There is also a `split track` button nearby.
